front/src/routes/integration/all/zwave/setup-page/actions.js

```diff
@@ -18,18 +18,20 @@ const addParamsAndFeatures = node => {
       const { min, max } = values[idx];
 
       if (values[idx].genre === 'user') {
-        const { category, type } = getCategory(values[idx]);
-        node.features.push({
-          name: `${values[idx].label} - ${node.product} -  Node ${node.id}`,
-          category,
-          type,
-          external_id: getDeviceFeatureExternalId(values[idx]),
-          read_only: values[idx].read_only,
-          unit: values[idx].units,
-          has_feedback: true,
-          min,
-          max
-        });
+        const { category, type } = getCategory(node, values[idx]);
+        if (category !== 'unknown') {
+          node.features.push({
+            name: `${values[idx].label} - ${node.product} -  Node ${node.id}`,
```

Comment:
Name may be free to fill by the user, isn't it ?

Reply:
Not for the feature, for now the approach we took is that this device-feature name is not used.